IbisLib/ibisplugin.h

```diff
@@ -37,6 +37,11 @@ class IbisPlugin : public QObject, public vtkObject
     QString GetPluginTypeAsString();
     virtual QString GetPluginDescription();
 
+    //Custom paths
```

Comment:
I don't think these functions are needed. Every plugin can easily get the instance of IbisAPI and call those functions directly.